src/gui/modalMenu.cpp

```diff
@@ -40,7 +41,8 @@ GUIModalMenu::GUIModalMenu(gui::IGUIEnvironment* env, gui::IGUIElement* parent,
 		m_menumgr(menumgr),
 		m_remap_dbl_click(remap_dbl_click)
 {
-	m_gui_scale = std::max(g_settings->getFloat("gui_scaling"), 0.5f);
+	const double screen_dpi_scale = RenderingEngine::getDisplayDensity();
```

Comment:
```suggestion
	const float screen_dpi_scale = RenderingEngine::getDisplayDensity();
```

Reply:
yep thanks it does only need to be a float